tokio/src/time/driver/sleep.rs

```diff
@@ -57,6 +57,7 @@ pub fn sleep_until(deadline: Instant) -> Sleep {
 /// [`interval`]: crate::time::interval()
 // Alias for old name in 0.x
 #[cfg_attr(docsrs, doc(alias = "delay_for"))]
+#[doc(alias = "wait")]
```

Comment:
You need to use `cfg_attr` to only set the alias when building documentation. This is because aliases are not supported on our minimum supported rust version.
```suggestion
#[cfg_attr(docsrs, doc(alias = "wait"))]
```